media/css/new-start-style.css

```diff
@@ -0,0 +1,299 @@
+    --text-connect: #D9A8FF;
+    --text-support: #85C2FF;
+    --text-get-involved: #FFB6F3;
+    --text-bug: #ABF69B;
+    --text-crash: #B0FFFF;
+    --text-warning: #FFB6F3;
+
+    --shadow: 0 4px 16px rgba(0, 0, 0, 0.3);
+  }
+}
+
+html {
+  font-family: system-ui, Open Sans, sans-serif;
+  background-color: var(--bg);
+  background-image: var(--bg-gradient),
+}
+
+body {
+  background-image: var(--logo-wireframe);
+  background-position: bottom right;
+  background-repeat: no-repeat;
+  color: var(--text);
+  margin: 0;
+  padding: 0;
+  min-height: 100vh;
+  max-width: 100%;
+  line-height: 1.6;
+}
+
+main {
+  display: grid;
+  gap: 2rem;
+  padding: 1rem;
+  margin: 0 auto;
+  max-width: 100ch;
+}
+
+header {
+  display: grid;
+  gap: 2rem;
+}
+
+svg {
+  fill: currentColor;
+  vertical-align: -4px;
+  margin-right: 0.25rem;
+}
+
+.header-top {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  align-items: center;
+}
+
+.logo-img { background-image: url('/media/img/thunderbird/logos/release-100.png'); }
+html[data-channel="earlybird"] .logo-img { background-image: url('/media/img/thunderbird/logos/earlybird-100.png'); }
+html[data-channel="beta"] .logo-img { background-image: url('/media/img/thunderbird/logos/beta.png'); }
+html[data-channel="daily"] .logo-img { background-image: url('/media/img/thunderbird/logos/daily-100.png'); }
+
+.logo-img {
+  display: inline-block;
+  position: relative;
+  background-size: contain;
+  background-repeat: no-repeat;
+  content: '';
+  width: 75px;
+  height: 75px;
+  align-self: center;
+  filter: drop-shadow(var(--shadow));
+}
+
+.title {
+  display: flex;
+  gap: 1rem;
+  margin: 0;
+  padding: 0;
+}
+
+.title-text {
+  font-size: 1rem;
+  font-weight: 300;
+}
+
+.tb-text {
+  font-size: 2rem;
+  font-weight: 400;
+  color: var(--text-logo);
+}
+
+.beta-text {
+  color: transparent;
+  background: var(--beta-gradient);
+  background-clip: text;
+  text-transform: uppercase;
+  font-weight: 600;
+}
+
+.donate {
+  display: grid;
+  gap: .25rem;
+  justify-self: end;
+  text-align: right;
+}
+
+.donate-btn {
+  display: block;
+  text-decoration: none;
+  justify-self: end;
+  background-color: #ff0039;
+  background-image: linear-gradient(#FF555F, #D70022);
+  border-radius: .25rem;
+  color: white;
+  padding: .5rem 1.25rem;
+  text-transform: uppercase;
+  font-weight: 700;
+  box-shadow: var(--shadow);
+  letter-spacing: .075rem;
+}
+
+.warning-panel {
+  display: grid;
+  place-items: center;
+  background: var(--bg-warning);
+  color: var(--text-warning);
+  border: 1px solid var(--text-warning);
+  border-radius: .5rem;
+  padding: 0.5rem 1rem;
+}
+
+.uppercase {
+  text-transform: uppercase;
+}
+
+.beta-thanks {
+  font-size: 1.25rem;
+}
+
+.main-content {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 4rem;
+}
+
+.warning-panel ul {
+  padding-left: 2.5rem;
+}
+
+.warning-panel a,
+.main-content a {
+  text-decoration: none;
+  position: relative;
+  font-weight: 700;
```

Comment:
For these links, since they all come with an SVG, we can add this style:
```
display: inline-flex;
align-items: center;
gap: 3px;
```

With this, you can remove the manually tweaked vertical alignment and the `span` element around the SVG if it's not needed